src/main/java/com/example/music_player/service/SourceService.java

```diff
@@ -59,7 +59,8 @@ public Source save(MultipartFile multipartFile, Song song, Long songIdFromDB) {
         Source source = Optional.ofNullable(sourceRepository.findByNameAndStorageType(name, storage_type, file_type))
                 .orElseThrow(() -> new IllegalStateException("source with " + name + " do not fined"));
         source.setStorage_types(storage_type);
-        return IOUtils.toByteArray(storageSourceService.findSongBySource(source));
+        final byte[] bytes = IOUtils.toByteArray(storageSourceService.findSongBySource(source));
```

Comment:
No need to introduce local variable just immediately return result of statement.